manifests/init.pp

```diff
@@ -1,112 +1,125 @@
-# == Class: simp
```

Comment:
Instead of one giant file, this should be split into subclasses so that they can be included independently of this if required. Each of the subclasses should be included based on a selector here and may use this class as a source of data.